pinot-common/src/test/java/org/apache/pinot/common/utils/RoundRobinURIProviderTest.java

```diff
@@ -48,7 +48,7 @@ public void testHostAddressRoundRobin()
         InetAddress.getByAddress("localhost", InetAddresses.forString("0:0:0:0:0:0:0:1").getAddress())
     };
 
-    MockedStatic<InetAddress> mock = Mockito.mockStatic(InetAddress.class);
+    try (MockedStatic<InetAddress> mock = Mockito.mockStatic(InetAddress.class)) {
```

Comment:
I may have been fooled by github, but isn't this breaking the indentation?

Reply:
Fixed.